test-requirements.txt

```diff
@@ -1,21 +1,11 @@
+-c https://raw.githubusercontent.com/ska-sa/katsdpdockerbase/master/docker-base-build/base-requirements.txt
+
 aioresponses==0.7.1
-attrs                           # via pytest
-coverage
-importlib-metadata              # via pytest
-more-itertools                  # via pytest
-msgpack                         # via katsdptelstate
-netifaces                       # via katsdptelstate
-packaging                       # via pytest
-pluggy                          # via pytest
-py                              # via pytest
 pytest
 pytest-asyncio==0.12.0
 pytest-cov
```

Comment:
Pytest-cov is not in `setup.cfg` (and the list there is not sorted alphabetically).

Reply:
It's not needed for users to run the unit tests (hence not in setup.py), but it's needed for CI which runs pytest with coverage.

Reply:
I've just came to the same conclusion :-D